collector/test/NetworkStatusNotifierTest.cpp

```diff
@@ -241,7 +238,14 @@ TEST(NetworkStatusNotifier, SimpleStartStop) {
    - we check that the former declared connection is deleted and redeclared as part of this network */
 TEST(NetworkStatusNotifier, UpdateIPnoAfterglow) {
   bool running = true;
-  MockCollectorConfig config;
+  setenv("ROX_AFTERGLOW_PERIOD", "0", 1);
+  CollectorArgs* args = CollectorArgs::getInstance();
+  int argc = 3;
+  const char* argv[] = {"collector", "--collection-method", "ebpf"};
+  int exitCode = 0;
+  args->parse(argc, const_cast<char**>(argv), exitCode);
+  CollectorConfig config(args);
+  unsetenv("ROX_AFTERGLOW_PERIOD");
```

Comment:
Same as the other cases. Afterglow is disabled using the ROX_AFTERGLOW_PERIOD environment variable instead of using a mock.